### PR TITLE
Fix/chainflip funding

### PR DESCRIPTION
**Pull request by 0xepicode**

### CODE REVIEW

File: packages/plugins/chainflip/src/broker.ts [hunk 1]
```diff
@@ -230,8 +229,9 @@ const fundStateChainAccount =
     evmToolbox: ReturnType<typeof ETHToolbox>;
     stateChainAccount: string;
     assetValue: AssetValue;
+    fromAddress: string;
   }) => {
-    if (assetValue.symbol !== "FLIP") {
+    if (assetValue.symbol !== "ETH.FLIP-0x826180541412D574cf1336d22c0C0a287822678A") {
```
Comment (towanTG) [inline packages/plugins/chainflip/src/broker.ts]:
```suggestion
    if (assetValue.toString() !== AssetValue.from({asset: "ETH.FLIP-0x826180541412D574cf1336d22c0C0a287822678A"}).toString()) {
```

Comment (towanTG) [reply packages/plugins/chainflip/src/broker.ts]: symbol will always fail, no?

Comment (towanTG) [reply packages/plugins/chainflip/src/broker.ts]: I would probably just create the assetValue for the asset we are testing here and then compare the to String value to be sure - doing this due to change to case sensitive addresses this week

Comment (0xepicode) [reply packages/plugins/chainflip/src/broker.ts]: good call, I ended up just using asset value and eq method

File: packages/plugins/chainflip/src/broker.ts [hunk 2]
```diff
@@ -230,8 +229,13 @@ const fundStateChainAccount =
     evmToolbox: ReturnType<typeof ETHToolbox>;
     stateChainAccount: string;
     assetValue: AssetValue;
+    fromAddress: string;
```
Comment (towanTG) [inline packages/plugins/chainflip/src/broker.ts]: this has to go :)

Comment (towanTG) [inline packages/plugins/chainflip/src/broker.ts]:
```suggestion
```

**Pull request merged**